src/main/java/org/folio/edge/rtac/utils/RtacUtils.java

```diff
@@ -0,0 +1,147 @@
+package org.folio.edge.rtac.utils;
+
+import static io.vertx.core.http.HttpHeaders.ACCEPT;
+import static org.folio.edge.core.Constants.APPLICATION_JSON;
+import static org.folio.edge.core.Constants.APPLICATION_XML;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.edge.core.utils.Mappers;
+import org.folio.edge.rtac.model.Instances;
+
+public final class RtacUtils {
+
```

Comment:
Personally, I would remove this blank line as I don't think it improves readability

Reply:
Done